common/app/views/fragments/inlineJSNonBlocking.scala.html

```diff
@@ -30,4 +30,32 @@
 // Ophan pageview ID and browser ID are needed by Google Analytics, which runs just after this script tag
 @InlineJs(ophanConfig().body, "ophanConfig.js")
 
+@if(WebpackSwitch.isSwitchedOn) {
+    @HtmlFormat.raw(common.Assets.js.curl)
+
+    [].slice.apply(document.querySelectorAll('figure.interactive[data-interactive]')).forEach(function (el) {
```

Comment:
Are we comfortable that this will be applied on all contentTypes, not tested to just articles/liveblogs? Also, if this is already in interactive articles, aren't you going to try and boot them twice?